server/package-lock.json

```diff
@@ -0,0 +1,1397 @@
+{
```

Comment:
Please **_delete_** the server folder and this file before merging in this PR. It is a remnant of the 'old ways'. Not sure why it is in here.